docs/guides/timelock-proposal.md

```diff
@@ -0,0 +1,333 @@
+# Timelock Proposal
+
+After adding FPS to your project dependencies, the next step is to create your
+first Proposal contract. In this example, we will create a proposal that deploys a new instance of `Vault.sol` and a new ERC20 token, then transfer ownership of both contracts to the timelock contract.
+
+```solidity
```

Comment:
What do we think about moving these code examples into the `examples/` directory?